provider/aws/aws.go

```diff
@@ -636,6 +661,7 @@ func (p *AWSProvider) submitChanges(ctx context.Context, changes Route53Changes,
 	}
 
 	var failedZones []string
+	debugLevel := log.DebugLevel
```

Comment:
Should we leave the log level as debug?

Reply:
This is not what it does, this not to change a debug level logging. What this line is basically doing, assigning to debugLevel variable a log level number of debug. The reason is, down the line, the log variable is shadowed in a loop

```
for z, cs := range changesByZone {
		log := log.WithFields(log.Fields{
			"zoneName": *zones[z].zone.N

```

so the only way I could do is to on line 708
```
log.Logger.IsLevelEnabled(debugLevel)
```

We could fix variable `log` shadowing, but not sure what the impact is going to be